docs/reST/ref/display.rst

```diff
@@ -78,12 +78,6 @@ required).
       Windows : windib, directx
       Unix    : x11, dga, fbcon, directfb, ggi, vgl, svgalib, aalib
 
-   On some platforms it is possible to embed the pygame display into an already
-   existing window. To do this, the environment variable ``SDL_WINDOWID`` must
-   be set to a string containing the window id or handle. The environment
```

Comment:
I think rather than deleting this, it should be updated to say that this is only for pygame 1.x and no longer applies to pygame 2.x due to the change from SDL 1 to SDL 2.